databricks_cli/clusters/cli.py

```diff
@@ -118,6 +124,14 @@ def list_cli(output):
 
     Returns information about all currently active clusters, and up
     to 100 most recently terminated clusters in the past 7 days.
+
+    By default the output format will be a human readable table with the following fields
+
+      - Cluster ID
```

Comment:
(minor) cluster ID

Reply:
Hm, what's the reason for this?

Reply:
nvm, this is just style. all we need is consistency.